nnvm/python/nnvm/top/nn.py

```diff
@@ -136,6 +143,8 @@ def schedule_conv2d(attrs, outs, target):
     with tvm.target.create(target):
         if groups == 1 and layout == "NCHW":
             return topi.generic.schedule_conv2d_nchw(outs)
+        elif groups == 1 and layout == "NCHW4c":
+            return topi.generic.schedule_conv2d_NCHWc_int8_prepacked(outs)
```

Comment:
@merrymercy I can't check dtype of input here. Could you comment here?

Reply:
Can we get dtype from outs[0].dtype?

Reply:
@merrymercy outs[0].dtype is a user input, which can be arbitrary